src/nodedef.h

```diff
@@ -115,9 +115,9 @@ struct NodeBox
 	// NODEBOX_FIXED
 	std::vector<aabb3f> fixed;
 	// NODEBOX_WALLMOUNTED
-	aabb3f wall_top;
-	aabb3f wall_bottom;
-	aabb3f wall_side; // being at the -X side
+	aabb3f wall_top{-1.0f, -1.0f, -1.0f, 1.0f, 1.0f, 1.0f};
```

Comment:
should maybe get a "historical mistake" comment too?

Reply:
reset actually changes the value to something well defined.
I'll change it to make that clearer